pkg/apis/certificates/types.go

```diff
@@ -43,6 +43,10 @@ type CertificateSigningRequestSpec struct {
 	// Base64-encoded PKCS#10 CSR data
 	Request []byte `json:"request"`
 
+	// signingProfile specifies a requested signing profile for a CA to use
+	// when issuing the certificate.
+	// SigningProfile string `json:"signingProfile,omitempty"`
```

Comment:
I don't actually mind the profiles existing for the approvers' benefit, but I'd expect that to be used more in status (e.g. "Approve CSR `foo` to be signed with profile `1-year-client`")